server/src/domain/facial-recognition/facial-recognition.service.spec.ts

```diff
@@ -95,8 +96,8 @@ const faceSearch = {
 
 describe(FacialRecognitionService.name, () => {
   let sut: FacialRecognitionService;
-  let assetMock: jest.Mocked<IAssetRepository>;
   let configMock: jest.Mocked<ISystemConfigRepository>;
+  let assetMock: jest.Mocked<IAssetRepository>;
```

Comment:
So close. This should just be how it was before :stuck_out_tongue:

Reply:
lol I just put it on line 99 without looking at the diff